.github/workflows/ci.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       - name: Build
         run: make build
       - name: Check clean repository
-        run: make check-clean-work-tree
+        run: make check-clean-work-tree verify-mods
```

Comment:
nit: I would rather add it to "Run linters" step as  we already have things like `verify-readmes` there.